### PR TITLE
[okhttp] Add fuzz target for LoggingInterceptor

### PR DESCRIPTION
**Pull request by aschaich**
Pending until #8617 is merged

### CODE REVIEW

File: projects/okhttp/src/main/java/ossfuzz/NullLogger.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 Google LLC
```
Comment (jonathanmetzman) [inline projects/okhttp/src/main/java/ossfuzz/NullLogger.java:1]: nit: wrong year.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Comment by jonathanmetzman** [comment]
Please resolve the conflicts and I'll merge

File: projects/okhttp/src/main/java/ossfuzz/NullLogger.java (hunk 1 above)
Comment (aschaich) [reply projects/okhttp/src/main/java/ossfuzz/NullLogger.java:1]: updated it

**Comment by aschaich** [comment]
rebased

**Pull request merged**